sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/Spark33XShims.scala

```diff
@@ -116,7 +116,7 @@ trait Spark33XShims extends Spark321PlusShims with Spark320PlusNonDBShims {
         ExprChecks.projectAndAst(
           TypeSig.astTypes,
           (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.MAP + TypeSig.ARRAY +
-              TypeSig.STRUCT + TypeSig.DECIMAL_128 + TypeSig.DAYTIME + TypeSig.YEARMONTH).nested(),
+              TypeSig.STRUCT + TypeSig.DECIMAL_128 + TypeSig.ansiIntervals).nested(),
```

Comment:
Please file a followup issue referencing this PR if not planning on removing these now unnecessary overrides in this PR.